src/unk_0203E880.c

```diff
@@ -887,7 +887,7 @@ UnkStruct_0203F478 * sub_0203F478 (FieldSystem * fieldSystem, int param1)
     return v0;
 }
 
-void sub_0203F598 (FieldSystem * fieldSystem)
+void FieldSystem_InitNewGameScripts (FieldSystem * fieldSystem)
```

Comment:
Took a look at script 9600 (which this invokes), which maps to file 412, script 0. From that, I would propose this name (and a little bit of cleanup of that trailing `return`):

```suggestion
void FieldSystem_InitNewGameState(FieldSystem *fieldSystem)
{
    sub_0203F5A4(fieldSystem, 9600);
}
```